app/posts/publish-teacher-training-courses/2021-06-03-placement-schools-and-training-centres.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Placement schools and training centres
+description: Asking providers to give more detailed location information
+date: 2021-06-14
+---
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+
+## The location problem
+
+Currently, Publish uses a model inherited from UCAS. This allows courses to show 2 types of location:
+
+* the training provider’s contact address
+their training locations – places a candidate
+* can apply to, but not necessarily where they will spend the majority, if any, of their time.
+
+Providers can add up to 37 addresses. Some set themselves up as multiple providers so they can add more locations (such as [West London Teaching School Alliance](https://web.archive.org/web/20210125160621/https://www.find-postgraduate-teacher-training.service.gov.uk/results/filter/provider?l=3&prev_l=none&prev_lat=none&prev_lng=none&prev_loc=none&prev_lq=none&prev_query=none&prev_rad=none&query=West+London+Teaching+School+Alliance)). Others only list one address (this is the case for 50% of SCITTs and 100% of HEIs).
+
+Listed addresses are not classified in any way (they could be administrative addresses, campuses, schools, hubs or areas).
+
+This is the [location problem](/publish-teacher-training-courses/the-location-problem); the information we have is inconsistent, which makes choosing courses based on location impossible for candidates.
+
+## What location data do we need?
+
+We’ve tried to use existing location data on Find, but it has proven problematic:
+
+* [Boosting HEI courses](/find-teacher-training/search-results-locations/) in search unfairly favoured one type of provider
+
+* [Travel areas](/find-teacher-training/searching-by-location/#a-new-approach) didn’t align with providers understanding of where they offer placements
+
+* [Showing contextual guidance](/find-teacher-training/searching-by-location/#improving-guidance) explaining how school placements work didn’t reflect individual provider working methods
+
+These changes and others were soon reverted due to unreliable data and differing provider behaviour.
+
+We can’t make improvements to Find until we have more accurate location data and can give providers the flexibility they desire.
+
+## What location data can providers give us?
+
+When we looked at locations from a candidate perspective, we considered [using Travel to Work Areas](/find-teacher-training/searching-by-location/#a-new-approach). This was based on the assumption that providers would be unable to give us more precise location information.
+
+When we spoke to providers to validate this, we discovered that providers maintain a list of placement or partner schools. We also found out that they have the URNs for these schools. A URN is a unique reference number used by the DfE for education establishments; using URNs means we can use structured data available from the [Get information about schools](https://www.get-information-schools.service.gov.uk) service.
```

Comment:
Suggestion: 'When we spoke to providers, they told us that they maintain a list of placement or partner schools.' 